src/System.Security.Cryptography.Primitives/src/System.Security.Cryptography.Primitives.TypeForwards.cs

```diff
@@ -0,0 +1,5 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Security.Cryptography.CryptographicException))]
```

Comment:
This shouldn't be needed. It should get copied from the ref (assuming it is a partial facade).